waku/wdd.md

```diff
@@ -0,0 +1,75 @@
+# Waku DNS Discovery
+
+> Version: 0.1.0 (Draft)
+>
+> Authors: Dean Eigenmann <dean@status.im>
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Specification](#specification)
+    1. [Records](#records)
+    2. [Traversing](#traversing)
+3. [Footnotes](#footnotes)
+4. [Copyright](#copyright)
+
+## Abstract
+
+In this specification, we describe the DNS discovery protocol used by [Waku](./waku.md). 
+
+## Specification
+
+This specification is based on the DNS discovery proposal for Ethereum<sup>1</sup>. It traverses a merkle tree that is stored in the DNS records of a chosen domain.
+
+### Records
+
+The node lists are encoded as merkle trees and stored within DNS TXT records. The root of the tree is a `TXT` record:
```

Comment:
Maybe this is obvious if you know exactly how DNS traversal etc works in more detail, but is this one or multiple requests? Should we specify this as a SHOULD or MAY?